contribs/gmf/externs/gmfx.js

```diff
@@ -27,6 +29,13 @@ gmfx.SearchDirectiveDatasource;
 
 
 /**
+ * The optional configuration for bloodhound for this set of data.
```

Comment:
You can change this to "Optional Bloodhound options for this data set." or "The optional Bloodhound configuration for this data set.". A link to the Bloodhound documentation would be good.